lib/vtls/gtls.c

```diff
@@ -1279,10 +1280,7 @@ gtls_connect_step3(struct connectdata *conn,
     #define use_addr in_addr
 #endif
     unsigned char addrbuf[sizeof(struct use_addr)];
-    unsigned char certaddr[sizeof(struct use_addr)];
-    size_t addrlen = 0, certaddrlen;
-    int i;
```

Comment:
where did you add this back

Reply:
Good catch, thanks! Seems like I removed it when amending the commit to move some variables again as Cadacy still complained. Doesn't the GnuTLS build on Travis treat implicit variable declarations as errors?